### PR TITLE
Feature loader, result counter

### PR DESCRIPTION
**Pull request by verbman**
Addition of loader to give searchinator a more prompt sense of interaction, included result counter to help users get an indication of the performance of their search 

# How to test this change

- [x] has automated tests
- [x] at least one a reviewer ran the code
- [x] tested in small screen
- [x] i18n
- [x] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

**Comment by Br3nda** [comment]
Approved. Dana is having a look at the code now. I ran and tested the functionality.

File: src/components/App.js [hunk 1]
```diff
@@ -35,15 +49,14 @@ class App extends Component {
...
           <AddressFinder data={this.props} />
           <button type="submit">Search</button>
-        </form>
-        <div>
-          {this.props.results.length > 0 &&
-            <button className="btn-toggle" onClick={() => {
-              this.setState({ showMap: !this.state.showMap}); }
-            }>{this.state.showMap ? 'Show List' : 'Toggle Map'}</button>
+          {this.props.itemsLoading && false &&
+            <i className="loader"><img src="loading.gif" alt="loading..." /></i>
```
Comment (dlouise64) [inline src/components/App.js]: @verbman is there any reason why we are using a gif here rather than font icon?

Comment (dlouise64) [inline src/components/App.js]: @verbman is there any reason why we are using a gif here rather than font icon?

File: src/styles/scss/Form.scss [hunk 2]
```diff
@@ -7,6 +7,39 @@ $primary-background: #fff;
...
+  width: 33px;
+  height: 33px;
+
+  img {
+    width: 33px;
+    height: 33px;
+  }
+}
+
+.loading {
```
Comment (dlouise64) [inline src/styles/scss/Form.scss]: @verbman Where abouts is `.loading` being used?

File: src/components/App.js (hunk 1 above)
Comment (verbman) [reply src/components/App.js]: I've removed those lines, we're still using the loading.gif but from css - my preference is for the gif being easier to maintain/implement

File: src/styles/scss/Form.scss (hunk 2 above)
Comment (verbman) [reply src/styles/scss/Form.scss]: See line 53 of App.js

**Pull request merged**